### PR TITLE
Add IFToMM 2018

### PR DESCRIPTION
**Pull request by octave-user**
Add link to [IFToMM 2018](https://www.researchgate.net/publication/327108298_Analysis_of_Analytical_Hydrodynamic_Bearing_Models_on_a_Reciprocating_Compressor_Vol_1)

### CODE REVIEW

File: Documentation/Publications.md [hunk 1]
```diff
@@ -57,185 +57,186 @@ _If your work uses **MBDyn**, please cite these papers as appropriate_:
...
-* William T. Yeager Jr., Raymond G. Kvaternik "**[Contributions of the Langley Transonic Dynamics Tunnel to rotorcraft technology and development](http://www.cs.odu.edu/~mln/ltrs-pdfs/NASA-aiaa-2000-1771.pdf)**" AIAA-2000-1771 AIAA/ASME/ASCE/AHS/ASC Structures, Structural Dynamics, and Materials Conference and Exhibit, 41st, Atlanta, GA, Apr. 3-6, 2000. 
-* Thomas Hablowetz "**Advanced Helicopter Flight and Aeroelastic Simulation based on General Purpose Multibody Code**" presented at the _AIAA Modeling and Simulation Technologies Conference and Exhibit_ Denver, CO, USA, August 14-17, 2000 (AIAA 2000-4299) 
+## Conference Papers:
+* E. P. Okabe, J. H. Izuka, R. Resch ["**Analysis of Analytical Hydrodynamic Bearing Models on a Reciprocating Compressor**"](https://www.researchgate.net/publication/327108298_Analysis_of_Analytical_Hydrodynamic_Bearing_Models_on_a_Reciprocating_Compressor_Vol_1) 10th International Conference on Rotor Dynamics – [IFToMM 2018](https://link.springer.com/chapter/10.1007/978-3-319-99262-4_22), Rio de Janeiro, 23-27 September 2018
```
Comment (mmorandi) [inline Documentation/Publications.md]: If I get it right, this paper uses OpenModelica, and no mention is made about MBDyn. Why cite it here? Does the implementation/model matches your work in MBDyn (likely yes, but perhaps we can explain the citation)?

Comment (octave-user) [reply Documentation/Publications.md]: Well, Eduardo Okabe used the old Fortran 90 code from module-hydrodynamic_plain_bearing for his paper. Actually, that's the only reason why I'm one of authors. In the meantime, the old Fortran code was converted to C++. Nevertheless, it is still the same algorithm.

Comment (octave-user) [reply Documentation/Publications.md]: OK, I have added this remark and link:
image

Comment (mmorandi) [reply Documentation/Publications.md]: Merged and published on www.mbdyn.org .
Thank you!

**Pull request merged**